src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -10,7 +10,7 @@
     private WorkWithFile workWithFile = new WorkWithFile();
 
     @Test
-    public void getStatisticAboutApple() {
+    public void getStatisticAboutApple() throws IOException {
```

Comment:
let's not modify test classes in any way

Reply:
fixed!